cloudfront/outputs.tf

```diff
@@ -1,3 +1,11 @@
 output "cloudfront_oai_iam_arn" {
   value = aws_cloudfront_origin_access_identity.origin_access_identity.iam_arn
 }
+
+output "cloudfront_domain_name" {
```

Comment:
Is it worth putting `s3` into the name of the output variable, just to make a little clearer?

Reply:
It's not really the s3 domain name, it's the domain name of the cloudfront distribution. I think it's the name of the distribution resource that's confusing things. I named it before I realised it was doing more than just s3 requests. So I would say leave this as is and rename `cloudfront_s3_distribution` to `cloudfront_distribution` What do you think?

Reply:
Sounds good to me